pinot-common/src/main/java/com/linkedin/pinot/common/data/TimeGranularitySpec.java

```diff
@@ -55,7 +65,18 @@ public TimeGranularitySpec(@Nonnull DataType dataType, @Nonnull TimeUnit timeTyp
     _timeType = timeType;
     _name = name;
   }
+  
+  public TimeGranularitySpec(@Nonnull DataType dataType, @Nonnull TimeUnit timeType, @Nonnull String timeFormat,
```

Comment:
Can you please add a javadoc and provide examples on what 'timeFormat' String values can be? thanks